core/drivers/sub.mk

```diff
@@ -19,3 +19,4 @@ srcs-$(CFG_DRA7_RNG) += dra7_rng.c
 srcs-$(CFG_STIH_UART) += stih_asc.c
 srcs-$(CFG_ATMEL_UART) += atmel_uart.c
 srcs-$(CFG_MVEBU_UART) += mvebu_uart.c
+srcs-$(CFG_SUNXI_UART) += sunxi_uart.c
```

Comment:
Are we sure we need new UART drive here, For instance, our pine64 port is re-using serial8250_uart.c
Why can't "serial8250_uart" be used here?